modules/ROOT/pages/configuring-mule-stacktraces.adoc

```diff
@@ -1,84 +1,86 @@
 
-Follow these steps to perform an autopsy for Mule in case it stops and you need
-more information to find the cause.
+Follow these steps to troubleshoot Mule if it stops and you need
+more information to find the cause:
 
-Obtain the Mule process ID, which we will refer to simply as `MULE_PID`. To identify
- the Mule process, use the Unix/Linux `ps` command, or instead execute the `jps`
-  command from the terminal. The jps tool ships with the JDK used to run Mule.
-  For example, a line containing "MuleContainerBootstrap" or "ApplicationDeployer"
-   or similar would identify the Mule process.
+. Identify the Mule process ID (`MULE_PID`).
+. Retrieve the thread dump.
+. Retrieve the heap dump if the Mule server throws
+ an Out Of Memory error: `java.lang.OutOfMemoryError: Java heap space`
+
+=== Identify the Mule process ID
+
+To identify the Mule process ID, use the Unix/Linux `ps` command, or execute the `jps`
+  command from the terminal. The `jps` tool ships with the JDK used to run Mule.
+  For example, a line containing `MuleContainerBootstrap`, `ApplicationDeployer`, or similar identifies the Mule process.
 
-=== Getting the Thread Dump
+=== Retrieve the Thread Dump
 
-Modern JDKs include the jstack tool, which you can utilize to get the thread dump:
+Use the `jstack` tool, which is available in modern JDKs, to retrieve the thread dump:
 
 ----
 jstack -l MULE_PID > filename.tdump
 ----
 
-=== Getting Heap Dump on Out of Memory Error
-
-Perform this procedure if you want to get a heap dump should the Mule server throw
- an Out Of Memory error: `java.lang.OutOfMemoryError: Java heap space`
+=== Retrieve the Heap Dump
 
-==== Procedure
+Retrieve the heap dump if the Mule server throws
+ an out of memory error: `java.lang.OutOfMemoryError: Java heap space`
 
-Oracle Java Virtual Machine provides a https://docs.oracle.com/javase/7/docs/webnotes/tsg/TSG-VM/html/clopts.html#gbzrr[JVM modifier]
- to gather this information. Note that you must restart the Mule server after
- applying this change, and reproduce the OOM error one more time.
+The Oracle JVM provides a https://docs.oracle.com/javase/7/docs/webnotes/tsg/TSG-VM/html/clopts.html#gbzrr[JVM modifier]
+ to gather this information. You must restart the Mule server after
+ applying this change and reproduce the out of memory error one more time.
 
-To apply this modifier to the JVM used by Mule, you can set a parameter in Mule's
- `/conf/wrapper.conf`, which automatically triggers the creation of a heap dump:
+To apply this modifier to the JVM that Mule uses, set the following parameters in Mule's `/conf/wrapper.conf` file to automatically trigger the creation of a heap dump:
 
 ----
 wrapper.java.additional.<n>=-XX:+HeapDumpOnOutOfMemoryError
 wrapper.java.additional.<n+1>=-XX:HeapDumpPath=path/to/dumps
 ----
 
-* change <n> to a number that follows the sequence in `wrapper.conf`
+When you set the parameters, Change <n> to a number that follows the sequence in `wrapper.conf`.
```

Comment:
I think the cases are wrong here, `change`?

Reply:
OK